modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/StandaloneCliOptions.cs

```diff
@@ -165,39 +171,65 @@ public class StandaloneCliOptions : Dictionary<string, string>, IAgentConfigProv
+                    { "csr", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("csr"); } },
+                    { "ab|applicationcertbase64=", "Legacy - do not use.",b => {isLegacyOption = true; legacyOptions.Add("ab|applicationcertbase64"); } },
+                    { "af|applicationcertfile=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("af|applicationcertfile"); } },
+                    { "pk|privatekeyfile=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("pk|privatekeyfile"); } },
+                    { "pb|privatekeybase64=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("pb|privatekeybase64"); } },
+                    { "cp|certpassword=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("cp|certpassword"); } },
+                    { "tb|addtrustedcertbase64=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("tb|addtrustedcertbase64"); } },
+                    { "tf|addtrustedcertfile=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("tf|addtrustedcertfile"); } },
+                    { "ib|addissuercertbase64=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("ib|addissuercertbase64"); } },
+                    { "if|addissuercertfile=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("if|addissuercertfile"); } },
+                    { "rb|updatecrlbase64=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("rb|updatecrlbase64"); } },
+                    { "uc|updatecrlfile=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("uc|updatecrlfile"); } },
+                    { "rc|removecert=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("rc|removecert"); } },
+                    { "dt|devicecertstoretype=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("dt|devicecertstoretype"); } },
+                    { "dp|devicecertstorepath=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("dp|devicecertstorepath"); } },
+                    { "i|install", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("i|install"); } },
+                    { "st|opcstacktracemask=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("st|opcstacktracemask"); } },
+                    { "sd|shopfloordomain=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("sd|shopfloordomain"); } },
+                    { "vc|verboseconsole=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("vc|verboseconsole"); } },
+                    { "as|autotrustservercerts=", "Legacy - do not use.", b => {isLegacyOption = true; legacyOptions.Add("as|autotrustservercerts"); } },
```

Comment:
Instead of having additional variable `isLegacyOption` and setting it to `true` for each deprecated option we can just check if `legacyOptions` list is empty or not.

Reply:
done